src/components/notifications/Toastr.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { Component } from "react";
+
+import { WithNamespaces, withNamespaces } from "react-i18next";
+import {
+  FaCheck,
+  FaExclamationCircle,
+  FaExclamationTriangle,
+  FaInfo
+} from "react-icons/lib/fa";
+import "./Toastr.css";
+
+const TIME_TO_SHOW = 5000;
+
+export type ToastrItem = {
+  id: number;
+  title: string;
+  description: string;
+  type: ToastrType;
+};
+
+export enum ToastrPosition {
+  topRight = "top-right",
+  bottomRight = "bottom-right"
+}
+
+export enum ToastrType {
+  success = "success",
+  warning = "warning",
+  error = "error",
+  info = "info"
+}
+
+enum ToastrBackground {
+  success = "#5cb85c",
+  warning = "#f0ad4e",
+  error = "#d9534f",
+  info = "#5bc0de"
+}
+
+type OwnProps = {
+  toastMessage: ToastrItem;
+  position?: ToastrPosition;
+  delay?: number;
+  onToastrClose: (event: ToastrItem) => void;
+  onErrorDetail: () => void;
+};
+type Props = WithNamespaces & OwnProps;
+
+type ToastrState = {
+  intervalId?: NodeJS.Timeout;
+  position: ToastrPosition;
+  toastList: ReadonlyArray<ToastrItem>;
+};
+
+class Toastr extends Component<Props, ToastrState> {
+  public initialState: ToastrState = {
+    position: ToastrPosition.topRight,
+    toastList: [],
+    intervalId: undefined
+  };
+
+  public state: ToastrState = {
+    toastList: [],
+    position: this.props.position
+      ? this.props.position
+      : ToastrPosition.topRight,
+    intervalId: undefined
+  };
+
+  public constructor(props: Props) {
+    super(props);
+    this.state = this.initialState;
+  }
+
+  public componentDidMount() {
+    this.setState({
+      toastList: [...this.state.toastList, this.props.toastMessage]
+    });
+    const intervalId = setInterval(
+      this.deleteToast,
+      this.props.delay || TIME_TO_SHOW
```

Comment:
`0` delay not supported
```suggestion
      fromNullable(this.props.delay).getOrElse(TIME_TO_SHOW)
```

Reply:
Fix with 710b28b